test/acceptance/framework/config/config.go

```diff
@@ -55,6 +55,11 @@ type TestConfig struct {
 func (t *TestConfig) HelmValuesFromConfig() (map[string]string, error) {
 	helmValues := map[string]string{}
 
+	// If Kind is being used they use a pod to provision the underlying PV which will hang if we
+	// use "Fail" for the webhook failurePolicy.
+	if t.UseKind {
+		helmValues["connectInject.failurePolicy"] = "Ignore"
```

Comment:
Let's use `setIfNotEmpty` to allow someone to override this value in tests
```suggestion
		setIfNotEmpty(helmValues, "connectInject.failurePolicy", "Ignore")
```

Reply:
good catch!